recipes/grblas/build.sh

```diff
@@ -0,0 +1,2 @@
+$PYTHON setup.py build_ext -I $PREFIX/include -L $PREFIX/lib
```

Comment:
Can you use the standard `pip install` command instead (see the example recipe)?